File: cpp/FizzBuzz/src/fizz_buzz.cpp
#include <string>

using namespace std;

string FizzBuzz(int number) {
    if (number % 3 == 0) {
        return "Fizz";
    }

    if (number % 5 == 0) {
        return "Buzz";
    }

    return to_string(number);
}

File: ruby/FizzBuzz/lib/FizzBuzz.rb
require "FizzBuzz/version"

module FizzBuzz

  class FizzBuzz

    def display(number)

      if number % 15 == 0
        return "FizzBuzz"
      end

      if number % 3 == 0
        return "Fizz"
      end

      if number % 5 == 0
        return "Buzz"
      end

      number.to_s
    end
  end

end
